shumai/tensor/tensor_ops.ts

```diff
@@ -16,7 +16,7 @@ export function relu(tensor: Tensor): Tensor {
   return tensor.maximum(scalar(0))
 }
 
-export function leakyRelu(tensor: Tensor, negative_slope: number): Tensor {
+export function leakyRelu(tensor: Tensor, negative_slope = 1e-3): Tensor {
```

Comment:
fine to sneak this in, but can you annotate with the type? `negative_slope: number = 1e-3`

Reply:
`bun format` removes the type annotation when the arg has a default

Reply:
:o @cryptodeal is that expected?